jgrapht-core/src/test/java/org/jgrapht/perf/flow/MaximumFlowAlgorithmPerformanceTest.java

```diff
@@ -37,7 +37,6 @@
 
```

Comment:
Yes this somehow sneaked in. Not sure whether (and how) I can take it out though.